source/retryable-writes/tests/README.rst

```diff
@@ -175,6 +175,26 @@ Each YAML file has the following keys:
       - ``data``: The data that should exist in the collection after the
         operation has been run.
 
+Split Batch Tests
+=================
+
+The YAML tests specify bulk write operations that are split by command type
+(e.g. sequence of insert, update, and delete commands). Multi-statement write
+operations may also be split due to ``maxWriteBatchSize`` or
+``maxBsonObjectSize``. For instance, an insertMany operation with three 9 MB
+documents would be split into three insert commands in order to respect the 16
```

Comment:
This is not necessarily true of drivers that are using OP_MSG on 3.6. It might be worth mentioning that OP_MSG supports write commands up to `maxMessageSizeBytes` (48000000 bytes). Drivers that split write command documents into OP_MSG payload 1 MUST account for the larger message size for this test.

Reply:
Thanks for the OP_MSG confirmation. I wasn't sure how that applied before, but I was only thinking about OP_QUERY.